container/apptabs.go

```diff
@@ -89,7 +90,7 @@ func (c *AppTabs) CreateRenderer() fyne.WidgetRenderer {
 	c.BaseWidget.ExtendBaseWidget(c)
 	r := &appTabsRenderer{line: canvas.NewRectangle(theme.ShadowColor()),
 		underline: canvas.NewRectangle(theme.PrimaryColor()), container: c}
-	r.updateTabs()
+	r.updateTabs(c.isTransitioning)
```

Comment:
When creating a renderer isn't this always false?

Reply:
Yes, `c.isTransitioning` would ideally always be `false` at this point. Would it be better if I explicitly set it as false?